.github/ISSUE_TEMPLATE/donation.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Donation Proposal
+about: Request membership in a OpenTelemetry Org
+title: 'Donation proposal: <name of donation>'
+labels: ''
+assignees: ''
+
+---
+
+<!-- Please remember to change the title of this issue by replacing
+ <name of donation> with the actual donated project name -->
+
+### Description of donated project
+
+TODO
+
+### Summary of existing usage of donated project
+
+TODO
+
+### Benefits to the OpenTelemetry community
+
+TODO
```

Comment:
`#### What prevents these benefits from being realized by keeping the donation as a separate project`?

Reply:
I did something...